cmake/sip_helper.cmake

```diff
@@ -7,7 +7,6 @@ set(__PYTHON_QT_BINDING_SIP_HELPER_DIR ${CMAKE_CURRENT_LIST_DIR})
 
 set(Python_ADDITIONAL_VERSIONS "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}")
 find_package(PythonInterp "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" REQUIRED)
-assert(PYTHON_EXECUTABLE)
```

Comment:
Why was this check removed?

Reply:
I get an 'Unknown CMake command "assert".'